### PR TITLE
Add Footer

### LINKED ISSUE #16: Add footer to the pod website

### PR DESCRIPTION
**Pull request by amalaabraham**
Resolves #16 

Added Footer

screenshot

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -8,7 +8,15 @@
         integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
         crossorigin="anonymous"></script>
     <script src="{{ site.baseurl }}/assets/js/hero.js"></script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
```
Comment (sachsom95) [inline _includes/header.html]: minor nit picks can u do bit of linting to make the code spaces more consistent

File: _includes/header.html [hunk 2]
```diff
@@ -8,7 +8,15 @@
...
+    integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css" integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w==" crossorigin="anonymous" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
```
Comment (sachsom95) [inline _includes/header.html:22]: remove additional space in between and at end of file

File: _sass/footer_minimal.scss [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+$font_color:#686b6f;
+
+#social_icon {
+    color: #686b6f;
+}
+// property for <hr>  for footer
+#break_line{
+//  background-color: $font_color;
+    border: none;
+     width: 100%;
```
Comment (sachsom95) [inline _sass/footer_minimal.scss]: Consistent spacing issue here and could remove the blank lines at top (minor nit picks :))

**Review by sachsom95 — COMMENTED** [review]
I will approve it asap if you can do minor style changes :)

**Comment by amalaabraham** [comment]
> I will approve it asap if you can do minor style changes :)

@sachsom95 Made the necessary changes. Have a look :)

**Review by sachsom95 — APPROVED** [review]
looks k ok :)

**Comment by sachsom95** [comment]
squash and merge used

File: index.html [hunk 4]
```diff
@@ -11,14 +11,14 @@
     
     {% include pod_leader.html %}
 
-    {% include members.html %}
+  <!-- {% include members.html %} -->
```
Comment (sachsom95) [inline index.html:14]: Why was the members.html changed @amalaabraham @suhanichawla any idea I just noticed it now

Comment (amalaabraham) [reply index.html:14]: Oops! Uncommented and made a PR #28

**Pull request merged**